padertorch/ops/losses/regression.py

```diff
@@ -49,16 +49,23 @@ def log_mse_loss(estimate: torch.Tensor, target: torch.Tensor,
             TasNet: A Dissecting Approach.” ArXiv:1911.08895
             [Cs, Eess], November 20, 2019.
             http://arxiv.org/abs/1911.08895.
+
+    >>> estimate = [[1., 2, 3], [4, 5, 6]]
+    >>> target = [[2., 3, 4], [4, 0, 6]]
+    >>> log_mse_loss(torch.tensor(estimate), torch.tensor(target))
+    tensor(0.9208)
+    >>> log_mse_loss(torch.tensor(estimate), torch.tensor(target), reduction=None)
+    tensor([0.0000, 0.9208])
     """
     # Use the PyTorch implementation for MSE, should be the fastest
     return _reduce(
-        F.mse_loss(estimate, target, reduce='none').mean(dim=-1).log10(),
-        reduce=reduce
+        F.mse_loss(estimate, target, reduction='none').mean(dim=-1).log10(),
+        reduction=reduction
     )
 
 
 def sdr_loss(estimates: torch.Tensor, targets: torch.Tensor,
-             reduce: str = 'mean'):
+             reduction: str = 'mean', axis=-1):
```

Comment:
Call this `dim` to also follow PyTorch?

Reply:
Good point. Most pytorch functions support both.
But this change was not intended. I removed the axis argument.